lib/client-options.js

```diff
@@ -20,7 +20,7 @@ function defaultOptions () {
       addressResolution: new addressResolution.AddressTranslator()
     },
     queryOptions: {
-      consistency: types.consistencies.one,
+      consistency: types.consistencies.localQuorum,
```

Comment:
This should probably be noted in the notes since it is a breaking change, and/or associated with a major version bump.